src/core/tiledmesh/qgstiledmeshlayerrenderer.cpp

```diff
@@ -62,6 +70,41 @@ bool QgsTiledMeshLayerRenderer::render()
     mElapsedTimer.start();
   }
 
+  if ( mLayerBoundingVolume )
+  {
+    const QgsCoordinateTransform transform = QgsCoordinateTransform( mMeshCrs, renderContext()->coordinateTransform().destinationCrs(), renderContext()->transformContext() );
+    try
+    {
+      std::unique_ptr< QgsAbstractGeometry > rootBoundsGeometry( mLayerBoundingVolume->as2DGeometry( transform ) );
```

Comment:
out of curiosity - does this work with google's tiles where the root bounding box encompasses the whole globe?

Reply:
No -- we'll probably need to handle that situation as a special case. (It's especially important when we're calculating the layer extent in 2d)

Reply:
okay cool